dev-haskell/streamly-core/streamly-core-0.2.2.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_HACKAGE_REVISION=1
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Streaming, parsers, arrays, serialization and more"
+HOMEPAGE="https://streamly.composewell.com"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="debug dev folds has-llvm limit-build-mem +opt unfolds unliftio"
```

Comment:
I won't bother you with this but I'm going to change your ebuild to automatically change 'has-llvm' to 'llvm' for consistency across Gentoo, and I'm going to adjust hackport maybe to automatically drop 'has-' prefixes. Again, nothing I'm asking you to do here.